x-pack/plugins/siem/server/lib/detection_engine/signals/signal_rule_alert_type.ts

```diff
@@ -87,6 +96,7 @@ export const signalRulesAlertType = ({
         'alert',
         alertId
       );
+      // const list = await services. // maybe add lists here?
```

Comment:
I'd change this into a TODO or remove it the line. Double comments within comments looks a bit off to me.